_docs/4-polarfire-soc.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: splash
+title: "PolarFire SoC"
+permalink: /docs/polarfire-soc/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the PolarFire SoC Page."
+feature_row:
+  - image_path: /assets/images/terminals-linux-300x500.png
+    alt: "customizable"
+    title: "Linux Developers"
+    excerpt: "Explore Linux on PolarFire SoC hardware platforms with Yocto, Buildroot and more."
+    url:  "/docs/linux-user/"
+    btn_class: "btn--primary"
+    btn_label: "Getting Started"
+  - image_path: /assets/images/bare_metal_developer-300x500.png
+    alt: "fully responsive"
+    title: "Bare Metal Developer"
```

Comment:
Should this be "Bare Metal Developers" (plural) in line with Nitin's "Linux Developers" & "FPGA Developers" comments?

Reply:
Yes it should be. I have made that change.